tests/util/taikoHelper.js

```diff
@@ -118,12 +119,22 @@ async function validateFormFromFile(configurations) {
     }
 }
 
+async function generateRandomDiagnosis(diseaseFile){
+    const jsonData = JSON.parse(fileExtension.parseContent(diseaseFile))
+    const snomedDiagnosesArray = jsonData.expansion.contains
+    const randomIndex = Math.floor(Math.random() * snomedDiagnosesArray.length);
+    const randomDiagnosis = snomedDiagnosesArray[randomIndex];
+    const randomDiagnosisData = randomDiagnosis.display;
```

Comment:
just storing the Display name, "randomDiagnosisData" variable name is not the right one